create_config.py

```diff
@@ -31,6 +33,12 @@ def main(file_path: Path):
     # Check for GC disc image
     if int.from_bytes(file_bytes[0x1C:0x20], byteorder="big") == 0xC2339F3D:
         create_gc_config(file_path, file_bytes)
+        return
+
+    psx_exe_header = file_bytes[0:8].decode()
```

Comment:
decode() tries to decode as utf-8, which can throw an exception. Better to not decode() and instead compare against `b"PS-X EXE"`. Or `decode("latin1")`, but that's kind of a hack

Reply:
I would vote for comparing against the bytes literally